### DISCUSSION TITLE
why I call the api as the api docs, but it failed

### DISCUSSION THREAD

**Discussion by mayanghong** [vercel/community#5759] [Bug Report]
### Page to Investigate

https://vercel.com/docs/rest-api/endpoints#create-a-team

### Steps to Reproduce

 url:https://api.vercel.com/v1/teams, code:400, err:{Code:payment_method_required Message:A payment method is required to create a team, please try creating a team again with a new payment method.

**Comment by leerob** [comment]
Hey there, have you already attached a payment method to your account?

**Reply by mayanghong** [reply]
how to add a payment method to my account, i have already create a pro team with credit card, but i can not find where i can associate the credit card with my account, it just relates team

**Reply by mayanghong** [reply]
And when I reactivate the pro team, it seems to error , errs message: "Your default payment method was not accepted. Please add a new payment method and set it as the default."
